Adopters.md

```diff
@@ -0,0 +1,13 @@
+# OpenEBS Dynamic NFS Volume Provisioner
+
+This is the list of organizations and users that publicly shared details of how they are using OpenEBS Dynamic NFS Volume Provisioner for running their applications. Please send PRs to add or remove organizations/users.
+
+
+The list of organizations that have publicly shared the usage of OpenEBS Dynamic NFS Volume Provisioner.
+
+| Organization | Stateful Workloads | Success Story |
+| :--- | :--- | :--- |
+|              |                    |               |
+
+
+You can see the full list of organizations/users using OpenEBS [here](https://github.com/openebs/openebs/blob/master/ADOPTERS.md).
```

Comment:
Till we get to beta, let us just include the text as follows:

```
You can see the full list of organizations/users using OpenEBS [here](https://github.com/openebs/openebs/blob/master/ADOPTERS.md).

If you are using Dynamic NFS, and would like yourself to be listed in this page, please raise an [issue](https://github.com/openebs/dynamic-nfs-provisioner/issues/new?assignees=&labels=&template=become-an-adopter.md&title=%5BADOPTER%5D) with the following details:

- Stateful Applications that you are running on OpenEBS
- Are you evaluating or already using in development, CI/CD, production
- Are you using it for home use or for your organization
- A brief description of the use case or details on how OpenEBS is helping your projects.


If you would like your name (as home user) or organization name to be added to the Adopters.md, please provide a preferred contact handle like github id, twitter id, linkedin id, website etc.

Reply:
Updated